### PR TITLE
Sen 103

### PR DESCRIPTION
**Pull request by NElnour**

### CODE REVIEW

File: app/src/App.vue [hunk 1]
```diff
@@ -63,7 +62,8 @@ export default {
       menuItems: [
         { title: "Inventory", path: "/inventory" },
         { title: "Orders", path: "/orders" },
-        { title: "History", path: "/history" }
+        { title: "History", path: "/history" },
```
Comment (JonathanYcWang) [inline app/src/App.vue]: we dont need this, we wont be doing history for the mvp deliverable

File: app/src/components/FoodBankOrdersTable.vue [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        this.foodbankOrders[index] = order;
+      });
+    }
+  },
+  data() {
+    return {
+      expanded: [],
+      foodbankOrders: [],
+      fields: [
+        { text: "EDI Number", value: "id", align: "center" },
```
Comment (JonathanYcWang) [inline app/src/components/FoodBankOrdersTable.vue]: an order id is not an edi number

File: app/store/index.js [hunk 3]
```diff
@@ -13,10 +13,15 @@ export default new Vuex.Store({
 	state: {
 		id: null,
 		inventoryItems: [],
-		activeOrders: []
+		activeOrders: [],
+		foodbankOrders: []
```
Comment (JonathanYcWang) [inline app/store/index.js]: why create the additional foodbankOrders array ?

File: app/src/App.vue (hunk 1 above)
Comment (kwpark23) [reply app/src/App.vue]: I didn't add it there, but I'll delete it

**Pull request merged**